src/Utils/Mappings.cs

```diff
@@ -42,9 +46,83 @@ public static class Mappings {
                 ["data"] = userData
             };
 
+            // 1136518 is 'Vilket är ditt prisförslag'
+            // 1136520 is 'Ange konsultens tillgänglighet'
+            var questionData = new JsonObject
+            {
+                ["data"] = new JsonArray
+                {
+                    new JsonObject
+                    {
+                        ["id"] = "1136518",
```

Comment:
Another way to for magic numbers, could possibly been:
```cs
private const string VilketArDittPrisForslag = "1136518";
private const string AngeKonsultensTillganglighet = "1136520";
```
And used them instead, so one doesn't have to randomly find the comments, but whatever. :)